packages/node/src/actions/byok/cancelOrder.ts

```diff
@@ -0,0 +1,54 @@
+import {
```

Comment:
a high level comment: I don't think it's productive to redefine all these code paths _just_ to express a different flow for signing the commitment. I'd suggest we refactor our approach here to be:
1. Operate on the wallet and fetch a commitment -- in this case you'd remove the order and fetch a commitment to the new wallet
2. Sign that commitment; where if the configuration requires an externally managed key we just sign using a different path.
3. Use the same path for submitting a cancel request to the relayer.

That way steps 1 and 3 re-use codepaths and step 2 expresses exactly the differentiated behavior we want. We wouldn't need to duplicate every action and rust helper this way

Reply:
I agree with these steps as they were what I had in mind when redefining these code paths: I thought this interface provided a better abstraction layer for both internally and externally managed keys since they do not care about how the wallet seeds and secrets were derived, only that they are provided. Therefore my plan was to refactor the old codepaths to be thin wrappers around these new functions, thus accomplishing steps 1-3.